src/toil_scripts/gatk_germline/germline.py

```diff
@@ -42,10 +42,11 @@
 
 import yaml
 from toil.job import Job
+from toil_lib import require
```

Comment:
1 line of whitespace between `toil` and `toil_lib` imports? (I thought that was the policy, but can't remember)